index.html

```diff
@@ -400,7 +401,7 @@ <h4 id="GitHub">GitHub</h4>
     <div class="left" style="width: 48%">
       <h4 id="roles">Roles</h4>
       <ul>
-        <li><a href="chair/role.html">Chair's role</a>; <a href="https://www.w3.org/wiki/MultipleChairs">Guidance
+        <li><a href="https://www.w3.org/wiki/MultipleChairs">Guidance
```

Comment:
Actually, MultipleChairs points to chair/role.html as a starting point...

Reply:
In my reading (and now re-reading) I see the two documents as pretty mixed. I see the first section of wiki/Multiple_Chairs as a better summary of a (single or multiple) chair's responsibilities than anything in chair/role.html

Reply:
So as a flow for a new person, I'd like to see

1. The summary of chair responsibilities
2. More detail as in chair/role.html
3. Considerations for multiple chairs (the rest of wiki/Multiple_Chairs)

Reply:
seems like it would make sense to merge the two documents into one - the role of the chair(s), describing the things they do. I am not sure the historical arguments leading to the increasingly common practice of having co-chairs are as interesting, but as a footnote it makes sense not to lose them.

If I propose edits, how do they get managed?

Reply:
I'll review them and merge if they make sense. If I need other opinions, I'll add them in the loop.